app/presenters/vm_presenter.rb

```diff
@@ -0,0 +1,12 @@
+class VmPresenter
+  def initialize(controller, session)
+    @controller = controller
```

Comment:
In reading [here](http://eewang.github.io/blog/2013/09/26/presenting-the-rails-presenter-pattern/) and [here](http://robertomurray.co.uk/blog/2014/decorators-presenters-delegators-rails/) and [here](http://mikepackdev.com/blog_posts/31-exhibit-vs-presenter), this object should be initialized with a `Vm`, but instead, is getting a controller and session.  It is not using the controller and I don't think it should be using the session.  What does `session[:tl_xml_blob_id]` represent?

@tenderlove @matthewd @Fryguy Please comment on how we should use Presenters.